source/linux/Foundational_Components/Power_Management/pm_low_power_modes.rst

```diff
@@ -290,6 +291,126 @@ system and it will go through a normal Linux boot process.
+   writes can be used to enable wakeup.
+
+   .. rubric:: Following commands can be used to enable wakeup from mcu_mcan0, mcu_mcan1,
+               mcu_uart0 and wkup_uart0 respectively.
+
+   .. code-block:: console
+
+      root@<machine>:~# devmem2 0x4084038 0x20050000
+      root@<machine>:~# devmem2 0x4084040 0x20050000
+      root@<machine>:~# devmem2 0x4084014 0x20050000
+      root@<machine>:~# devmem2 0x4084024 0x20050000
+
+   .. note::
+
+      Atleast one of the wakeup sources listed above must be enabled to wakeup from I/O Only Plus DDR mode.
+
+   .. rubric:: To enter I/O Only Plus DDR mode, first disable wakeup from RTC, USB0 and USB1 as these wakeup
+               sources are not supported for this mode.
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62AX')
+
+      .. code-block:: console
+
+         root@am62axx-evm:~# echo disabled > /sys/class/rtc/rtc0/device/power/wakeup
+         root@am62axx-evm:~# echo disabled > /sys/devices/platform/bus@f0000/f900000.dwc3-usb/power/wakeup
+         root@am62axx-evm:~# echo disabled > /sys/devices/platform/bus@f0000/f910000.dwc3-usb/power/wakeup
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62PX')
+
+      .. code-block:: console
+
+         root@am62pxx-evm:~# echo disabled > /sys/class/rtc/rtc0/device/power/wakeup
+         root@am62pxx-evm:~# echo disabled > /sys/devices/platform/bus@f0000/f900000.usb/power/wakeup
+         root@am62pxx-evm:~# echo disabled > /sys/devices/platform/bus@f0000/f910000.usb/power/wakeup
+
+   .. rubric:: Then, configure PMIC register bit to turn off only selected rails for this mode.
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62AX')
+
+      .. code-block:: console
+
+         root@am62axx-evm:~# i2cset -f -y -m 0xFF -r -a 0 0x48 0x86 0x1
+
+      The register write has been done to enable PMIC to enter `PMIC S2R <https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.ti.com/lit/pdf/slvucm3&ved=2ahUKEwjknrL864-KAxVocGwGHdJZFPwQFnoECBgQAQ&usg=AOvVaw1y0evgQxxy9YOF7l5Wzaef>`_ .
+
+   .. ifconfig:: CONFIG_part_variant in ('AM62PX')
+
+      .. code-block:: console
+
+         root@am62pxx-evm:~# i2cset -f -y -m 0xFF -r -a 0 0x48 0x86 0x2
+
+   .. rubric:: Now, the SoC can be suspended using the following command:
+
+   .. code-block:: console
+
+         root@<machine>:~# echo mem > /sys/power/state
+         [   26.132900] PM: suspend entry (deep)
+         [   26.136759] Filesystems sync: 0.000 seconds
+         [   26.151748] Freezing user space processes
+         [   26.157256] Freezing user space processes completed (elapsed 0.001 seconds)
+         [   26.164239] OOM killer disabled.
+         [   26.167469] Freezing remaining freezable tasks
+         [   26.173168] Freezing remaining freezable tasks completed (elapsed 0.001 seconds)
+         [   26.180624] printk: Suspending console(s) (use no_console_suspend to debug)
+
+   This partially indicates that linux has finished it's I/O Only plus DDR sequence.
```

Comment:
I think think this can be slightly misleading, the above log tells me nothing special that we are entering IO+DDR. This is the same for any suspend-to-mem scenario right?

Reply:
Exactly, that's why we added LED representation at the end

Reply:
What I meant was let's reword this sentence to not say "linux has finished" rather say:
```suggestion
   This indicates that the device has partially completed the I/O Only plus DDR entry sequence.
```